dockerfiles/alpine/is-analytics/Dockerfile

```diff
@@ -66,12 +70,21 @@ RUN mkdir -p ${USER_HOME}/.java/.systemPrefs && \
     chmod -R 755 ${USER_HOME}/.java && \
     chown -R ${USER}:${USER_GROUP} ${USER_HOME}/.java
 
-# copy wso2 product distribution zip files to user's home directory and set ownership
-COPY --chown=wso2carbon:wso2 ${FILES}/${WSO2_SERVER_PACK}/ ${USER_HOME}/${WSO2_SERVER_PACK}/
-# copy init script to user home
-COPY --chown=wso2carbon:wso2 init.sh ${USER_HOME}/
-# copy mysql connector jar to the server as a third party library
-COPY --chown=wso2carbon:wso2 ${FILES}/mysql-connector-java-*-bin.jar ${USER_HOME}/${WSO2_SERVER_PACK}/repository/components/lib/
+# add the WSO2 product distribution to user's home directory
+RUN \
+    wget --no-check-certificate -O ${WSO2_SERVER}.zip "${WSO2_SERVER_DIST_URL}" \
+    && unzip -d ${USER_HOME} ${WSO2_SERVER}.zip \
+    && chown wso2carbon:wso2 -R ${WSO2_SERVER_HOME} \
+    && mkdir ${USER_HOME}/wso2-tmp/deployment \
+    && cp -r ${WSO2_SERVER_HOME}/repository/deployment ${USER_HOME}/wso2-tmp/deployment \
+    && rm -f ${WSO2_SERVER}.zip
+
+# copy docker-entrypoint script to user home
+COPY --chown=wso2carbon:wso2 docker-entrypoint.sh ${USER_HOME}/
```

Comment:
@RidmiR shall we move this `COPY` instruction before the `RUN` instruction? This is since, the `docker-entrypoint.sh` script is less prone to changing than the product pack. We can move the instructions which we expect the remain static or less prone to changes before the instructions which are more prone to changing. Hence, we can avoid rebuilding such layers during a build, if they already exist in the local cache.